content/blog/2024/202402-03-review.rst

```diff
@@ -0,0 +1,112 @@
+==================================================================
+sciwork February and March 2024 review: CFD workshop and scisprint
+==================================================================
+
+:date: 2024-05-12
+:author: Ming-Jie Kuo
+:category: blog
+
+Weekly Meetup
+-------------
+During the meetup events in February and March, modmesh contributors made the installation process
+of modmesh on windows computers smoother and fixed many CI related issues, while some contributors
+developed many plotting related features to make the presentation of the mesh more flexible.
+
+In addition to project development, there were also many people having questions about their work.
+Some people asked about the difficulties they encountered when working on the Vue.js project and
+auto-streaming databases, and some people asked how to improve the performance of LLM models from
+both software and hardware perspectives.
+
+March Workshop
+--------------
+Sciwork organized a new event called Advanced CFD software development workshop in March. Chun-Hsu and
```

Comment:
Please keep sciwork lower-case everywhere.

Reply:
Miss one. I fixed it.